cmd/ocm-backplane/console/console_test.go

```diff
@@ -149,17 +147,20 @@ var _ = Describe("console command", func() {
 			checkCapturedCommands()
 		})
 	})
-*/
-// This test verifies that the console container is still started the same way after issuing a
-// 'oc project <namespace id>' command.
-//
-// In particular this test checks that the name of container started by the 'ocm backplane console'
-// command is based on the cluster id and not on the supposed cluster name extracted from kube config.
-// Issuing a 'oc project <namespace id>' will create a new context with a new cluster in kube config
-// - Which does not contain any bit of information concerning the OSD cluster name.
-// - Which contains ':' char which is an invalid char in a container name.
-
-/*
+
+	// This test verifies that the console container is still started the same way after issuing a
+	// 'oc project <namespace id>' command.
+	//
+	// In particular this test checks that the name of container started by the 'ocm backplane console'
+	// command is based on the cluster id and not on the supposed cluster name extracted from kube config.
+	//
+	// Indeed 'oc' client is actually connected to the hive cluster which proxy commands to the targeted
+	// OSD cluster.
+	// Issuing a 'oc project <namespace id>' will create a new context with a new cluster in kube config...
+	// but the name of the newly created cluster config will be based on the hive cluster URL:
+	// - Which does not contain any bit of information concerning the OSD cluster name.
+	// - Which contains ':' char which is an invalid char in a container name.
+
```

Comment:
setupConfig() [ line 113] should change with utils.CreateTempKubeConfig() method otherwise all tests may fail randomly 

https://github.com/openshift/backplane-cli/blob/1b6899874c404a9f365f06274563c3ffa3c99716/cmd/ocm-backplane/console/console_test.go#L113